Composer/packages/lib/ui-shared/src/components/Toolbar.tsx

```diff
@@ -64,7 +65,7 @@ const renderItemList = (item: IToolbarItem, index: number) => {
     return (
       <ActionButton
         key={index}
-        css={actionButton}
+        css={!item.buttonProps?.styles ? actionButton : null}
```

Comment:
can we include the `actionButton` style in item.buttonProps so we don't need to special case this?

Reply:
Do you mean, add the actionButton styles to every single Toolbar item across the app?

Reply:
I mean to include the `actionButton` css on the toolbar items coming in from props so you can remove this line altogether and have the `{...item.buttonProps}` spread include what is needed

Reply:
That's what I meant -- update *every toolbar item across the app* to have these styles inline rather than be the fallback default styles.

Reply:
ok I refactored this style a bit, and now it is passed in via props from the few places it is used.